### DISCUSSION TITLE
Cannot use copilot for Visual Studio 2022 behind proxy

### DISCUSSION THREAD

**Discussion by jhested** [community/community#39867] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hi,

When trying to run copilot in VS2022 I get the following exception 
`GitHub Copilot 1.61.0.0 (v1.61.0.0@5e1cdbba5)
GitHub Copilot Agent v1.61.0
[Error] An error occurred for 'RunOperation:SignInInitiateAsync': StreamJsonRpc.RemoteInvocationException: Request signInInitiate failed with message: self signed certificate in certificate chain
   at StreamJsonRpc.JsonRpc.<InvokeCoreAsync>d__143`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at GitHub.Copilot.CopilotClient.<RunOperation>d__17`1.MoveNext()
[Error] An error occurred for 'RunOperation:RunOperation': StreamJsonRpc.RemoteInvocationException: Request signInInitiate failed with message: self signed certificate in certificate chain
   at StreamJsonRpc.JsonRpc.<InvokeCoreAsync>d__143`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at GitHub.Copilot.CopilotClient.<RunOperation>d__17`1.MoveNext()
NotSignedIn
`

I got it working in VS Code by also installing the Win-CA extension and then changing Win-CA extension settings to append, but I cannot get it working in VS 2022.

Any help is much appreciated

**Comment by ossanna16** [comment]
Hi there @jhested and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
